pepper-apis/config/testing-inmemorydb.conf.ctmpl

```diff
@@ -30,6 +30,7 @@
     "dsmJwtSigner": "org.broadinstitute.kdux",
     "dbInstanceId": "ddp-dev",
     "restrictRegisterRoute": false,
+    "restrictAuth0LogEventRoute": false,
```

Comment:
@evgeniipr do we have the procedure for changing this to `true` documented in the release plan?

Reply:
@zyme actually not yet. Could you give me some example to write a release plan doc?

Reply:
This is just the local testing file so it'll be fine. The one used in real code is rendered from Vault.